packages/@aws-cdk/aws-docdb/lib/cluster.ts

```diff
@@ -146,6 +147,14 @@ export interface DatabaseClusterProps {
    * @default - false
    */
   readonly deletionProtection?: boolean;
+
+  /**
+   * The configuration of log types that can be enabled for exporting to Amazon CloudWatch Logs.
+   * You can enable audit logs or profiler logs.
+   *
+   * @default {}
+   */
+  readonly cloudwatchLogsExports?: CloudwatchLogsExportsProps;
 }
```

Comment:
Actually, let's get rid of the `CloudwatchLogsExportsProps` interface, and just have 2 separate properties in `DatabaseClusterProps`, `exportAuditLogsToCloudWatch?: boolean` and `exportProfileLogsToCloudWatch?: boolean`.

Reply:
it's ok

Reply:
changed this